plugins/volume/persistent_test.go

```diff
@@ -0,0 +1,208 @@
+package volume
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/taskcluster/slugid-go/slugid"
+	"github.com/taskcluster/taskcluster-worker/engines"
+	"github.com/taskcluster/taskcluster-worker/engines/extpoints"
+	_ "github.com/taskcluster/taskcluster-worker/engines/mock"
+	"github.com/taskcluster/taskcluster-worker/plugins"
+	pluginExtpoints "github.com/taskcluster/taskcluster-worker/plugins/extpoints"
+	"github.com/taskcluster/taskcluster-worker/plugins/plugintest"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+var logger, _ = runtime.CreateLogger(os.Getenv("LOGGING_LEVEL"))
```

Comment:
Doesn't seem to work when it's at the package level and not inside the body of a func.

plugins/volume/persistent_test.go:20:1: expected declaration, found 'IDENT' logger